vast.yaml.example

```
@@ -37,6 +37,8 @@ vast:
   max-taste-partitions: 5
   # The amount of queries that can be executed in parallel.
   max-queries: 10
+  # The false positive rate for lossy structures in the meta index.
+  meta-index-fprate: 0.01
```

```suggestion
  meta-index-fp-rate: 0.01
```

With the other way of writing, it is hard to see the word boundaries.